README.md

```diff
@@ -178,3 +179,9 @@ To make the integration simple we have developed proxy client SDKs. You can find
 - [JavaScript Proxy SDK (browser)](https://github.com/unleash-hosted/unleash-proxy-client-js)
 - [Android Proxy SDK](https://github.com/Unleash/unleash-android-proxy-sdk)
 - [iOS Proxy SDK](https://github.com/Unleash/unleash-proxy-client-swift)
+
+## OpenAPI
+
+The proxy exposes an optional OpenAPI spec and Swagger page. If you run it with the `ENABLE_OAS` (environment variable) or `enableOAS` (in-code configuration variable) set to `true`, these can be found at `<base url>/docs/openapi.json` and `<base url>/docs/openapi` respectively.
+
+The swagger page is an interactive page where you can discover and test the API endpoints the proxy exposes. The JSON spec can be used to generate an OpenAPI client with OpenAPI tooling such as the [OpenAPI generator](https://openapi-generator.tech/).
```

Comment:
Perhaps we should move this up so that the motivational text comes first, before the configuration details.

Reply:
Agree; I'll fix!